yellowbrick/regressor/residuals.py

```diff
@@ -502,6 +504,7 @@ def __init__(
         model,
         ax=None,
         hist=True,
+        qqplot=False,
```

Comment:
1. qqplot parameter should be added to the docstring.
2. The docstring should contain a description of what qqplot 
3. The docstring should include what are the available options for qqplot (ie. True, False)

Reply:
Thank you for pointing this out, I added qqplot to the docstring of ResidualsPlot.